modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java

```diff
@@ -104,7 +104,23 @@ public void updateUser(PatchOperationRequestObject patchUserInfo, String userId)
 
         try (CloseableHttpResponse response = getResponseOfHttpPatch(endPointUrl, jsonRequest, getHeaders())) {
             Assert.assertEquals(response.getStatusLine().getStatusCode(), HttpServletResponse.SC_OK,
-                    "Role update failed");
+                    "User update failed");
+        }
+    }
+
+    /**
+     * Search a user and get requested attributes
```

Comment:
Shall we add the missing full stop here.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096